auth0/authentication/back_channel_login.py

```diff
@@ -1,38 +1,61 @@
-from typing import Any
+from typing import Any, Optional, Union, List, Dict
 
 from .base import AuthenticationBase
 
+import json
+
 
 class BackChannelLogin(AuthenticationBase):
     """Back-Channel Login endpoint"""
 
     def back_channel_login(
-        self, binding_message: str, login_hint: str, scope: str, **kwargs
+        self,
+        binding_message: str,
+        login_hint: str,
+        scope: str,
+        authorization_details: Optional[Union[str, List[Dict]]] = None,
+        **kwargs
     ) -> Any:
         """Send a Back-Channel Login.
 
         Args:
              binding_message (str): Human-readable string displayed on both the device calling /bc-authorize and the user’s 
              authentication device to ensure the user is approves the correct request.
 
-             login_hint (str): String containing information about the user to contact for authentication.
+             login_hint (str):  JSON string containing user details for authentication in the iss_sub format.Ensure 
+             serialization before passing.
 
              scope(str): "openid" is a required scope.Multiple scopes are separated 
              with whitespace.
 
-             **kwargs: Other fields to send along with the PAR.
+             authorization_details (str, list of dict, optional): JSON string or dictionary representing
+             Rich Authorization Requests (RAR) details to include in the CIBA request.
+             If a Python list is provided, the SDK automatically serializes it to a JSON string before sending.
```

Comment:
> If a Python list is provided, the SDK automatically serializes it to a JSON string before sending.

IMO, this is just internal implementation details, there is no need to explain it